app/frontend/components/pages/accounts/accounts.tsx

```diff
@@ -0,0 +1,286 @@
+import {Fragment, h} from 'preact'
+import {connect} from '../../../helpers/connect'
+import actions from '../../../actions'
+import range from '../../../../frontend/wallet/helpers/range'
+import printAda from '../../../helpers/printAda'
+import {Lovelace} from '../../../state'
+import {AdaIcon} from '../../common/svg'
+import Alert from '../../common/alert'
+import SendTransactionModal from './sendTransactionModal'
+import DelegationModal from './delegationModal'
+import tooltip from '../../common/tooltip'
+
+const Account = ({
+  i,
```

Comment:
this seems weird - why do you need `i`, `selectedAccount` and `account` and what's the difference between them three? Isn't the account index contained within the `account` object?

Reply:
Fixed.